lib/matplotlib/colors.py

```diff
@@ -243,6 +243,18 @@ def _check_color_like(**kwargs):
             raise ValueError(f"{v!r} is not a valid value for {k}")
 
 
+def check_color_like_list(**kwargs):
```

Comment:
```suggestion
def _check_color_like_list(**kwargs):
```

Sorry @i-deal I should have spotted this before.  I think the idea is that this should be a private function, so its name should start with an underscore, consistent with the existing `_check_color_like`.  Obviously the test will need updating to match.